build/ci/github_actions/operator/save_driver_pods_logs.sh

```diff
@@ -5,11 +5,12 @@ get_all_pods_by_type (){
     kubectl get pod -l csi | grep $pod_type | awk '{print$1}'
 }
 
-run_action_on_pod (){
+save_action_output (){
```

Comment:
Why did you rename it? I think the main thing this function does is running the action.

Reply:
@zingero @oriyarde decide

Reply:
I can settle for "run_action_and_save_output"

Reply:
Me too